jsettlers.logic/src/main/java/jsettlers/logic/movable/Movable.java

```diff
@@ -53,11 +54,10 @@
  * @author Andreas Eberle
  *
  */
-public final class Movable implements IScheduledTimerable, IPathCalculatable, IDebugable, Serializable, IViewDistancable, IGuiMovable,
-		IAttackableMovable {
+public final class Movable implements ILogicMovable {
 	private static final long serialVersionUID = 2472076796407425256L;
-	private static final HashMap<Integer, Movable> movablesByID = new HashMap<Integer, Movable>();
-	private static final ConcurrentLinkedQueue<Movable> allMovables = new ConcurrentLinkedQueue<Movable>();
+	private static final HashMap<Integer, ILogicMovable> movablesByID = new HashMap<Integer, ILogicMovable>();
```

Comment:
I think we should stick to Movable here, as this is an internal map (same goes for allMovables). Only Movables can be stored here. What do you think?